src/views/matching/MissionWriteView.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+  <TopNavigation :showBack="true" :showLogoText="false" />
+  <div class="relative flex justify-center w-full">
+    <div class="bg-ivory flex flex-col gap-6 w-full mt-10 px-6 justify-between">
```

Comment:
```suggestion
  <div class="relative flex justify-center w-full">
    <TopNavigation :showBack="true" :showLogoText="false" />
    <div class="bg-ivory flex flex-col gap-6 w-full mt-10 px-6 justify-between">
```

이런 식으로 네비게이션 컴포넌트를 relative인 div 안에 넣어주면 버튼이 화면을 벗어나지 않아 더 통일성 있을 것 같아요!